src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -0,0 +1,185 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs";
+import ReactDOMServer from "react-dom/server";
+import { withDesign } from "storybook-addon-designs";
+
+import Tooltip from "./Tooltip";
+import Button from "../Button/Button";
+import Radio from "../Radio/Radio";
+import Icon from "../Icons/Icon";
+import Link from "../Link/Link";
+import Label from "../Label/Label";
+import Image from "../Image/Image";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("Tooltip")}
+  component={Tooltip}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36835%3A25702",
+    },
+    jest: ["Tooltip.test.tsx"],
+  }}
+  argTypes={{
+    className: { control: false },
+    id: { control: false },
+    isDisabled: {
+      table: { defaultValue: { summary: false } },
+    },
+  }}
+/>
+
+# Tooltip
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.0.9`    |
+| Latest            | `1.0.9`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Examples](#examples)
+
+## Overview
+
+<Description of={Tooltip} />
+
+The `Tooltip` component shows contextual help or information about a specific
+component when a user hovers or focuses on the component.
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="Tooltip with Controls"
+    args={{
+      content:
+        "This is a tooltip with information describing the hovered object.",
+      className: undefined,
+      id: "tooltip-id",
+      isDisabled: false,
+    }}
+  >
+    {(args) => (
+      <Tooltip
+        content={args.content}
+        className={args.className}
+        id={args.id}
+        isDisabled={args.isDisabled}
+      >
+        <Button id="tooltip-button">Hover me</Button>
+      </Tooltip>
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Tooltip with Controls" />
+
+## Accessibility
+
+The `Tooltip` component is accessible via keyboard. The color contrast between
```

Comment:
Add a new paragraph at `If a tooltip is associated…`

Add the following paragraph to the Accessibility section…

The DS `Tooltip` component is implemented with Chakra UI and Chakra handles the appropriate `aria-describedby` attribute for the element that the tooltip is associated with.